### ISSUE TITLE
What is the training procedure?

### ISSUE THREAD

**Issue by gaohuazuo** [lllyasviel/sketchKeras#4]
Can you briefly describe the dataset and training loss/algorithm used for the "sketchifying" neural network? I couldn't find any comments after reading this repo, paintschainer and your style2paints repo and paper. Many thanks.
